examples/pixart/controlnet_pixart_alpha.py

```diff
@@ -0,0 +1,292 @@
+from typing import Any, Dict, Optional
```

Comment:
I think maybe pipelines can go to the /example/community folder, the training script can stay in example/pixart folder

cc @sayakpaul

Reply:
Okay with that plan.

Reply:
This needs to be addressed first.